apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -885,8 +887,45 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
       outFolderPath = path.resolve(this._configuration.buildFolder, outFolderPath);
     }
 
+    outFolderPath = Path.convertToSlashes(outFolderPath);
+    outFolderPath = outFolderPath.replace(/(\/*)$/, '/'); // Ensure the outFolderPath ends with a slash
```

Comment:
Should we have a `Path.ensureTrailingSlash()`?

Reply:
Can outFolderPath have backslashes in it?

Reply:
What should the semantics of that be? Should there be a `Path.ensureTrailingBackslash`, or would we include that in an options object?